openassessment/xblock/test/test_submission.py

```diff
@@ -153,6 +153,55 @@ def test_cannot_submit_in_preview_mode(self, xblock):
         self.assertEqual(resp[1], "ENOPREVIEW")
         self.assertIsNot(resp[2], None)
 
+    @patch("openassessment.xblock.openassessmentblock.import_student_module")
```

Comment:
Are we testing what should happen when a student is not allowed to reset their submission?

Reply:
That would mean for this call to return false: https://github.com/openedx/edx-ora2/pull/2187/files#diff-02c29fde1ca6f6433709a901d2d3c9ebe59de8f5c7dfde912c0ffdd3f86e95deR181-R186

Reply:
If the student cannot reset the submission, the button does not appear, and therefore the handler will never be called. All tests that test the `allow_resubmission` function are in the `test_allow_resubmission.py` module.